streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -47,6 +47,7 @@
     private final ArrayDeque<ConsumerRecord<byte[], byte[]>> fifoQueue;
 
```

Comment:
Just read the class JavaDoc:
```
It also keeps track of the partition timestamp defined as the minimum timestamp of records in its queue
```

That is not correct any more. Can we update it?

Reply:
Yep